3landers.sol

```diff
@@ -0,0 +1,780 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/finance/PaymentSplitter.sol";
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/token/ERC721/extensions/ERC721Enumerable.sol";
```

Comment:
in general it's good to check the diff of these imports with the source to make sure they're the same, for the purposes of this review i'll just assume they're fine